ReadMe.md

```diff
@@ -38,10 +41,9 @@ setTimeout(function (msg) {
 }, 5000, 'Hello world');
 ```
 
-**Timer setTimeoutAt(Function callback, number timestamp, mixed arg1, mixed arg2, ...)**
+**Timeout setTimeoutAt(Function callback, number timestamp, mixed arg1, mixed arg2, ...)**
 
-Calls `callback` when our clock reaches the given `timestamp` (in milliseconds). All arguments passed after the
-`interval` will be passed to the callback once it gets invoked. Returns a `Timer` instance.
+- Creates and returns a `Timeout` instance that will call `callback` when our clock reaches the given `timestamp` (in milliseconds). All arguments passed after the `interval` will be passed to the callback once it gets invoked.
```

Comment:
I would recommend losing the `- ` characters at the start of this line as they are causing bulleted list formatting for the description of the method. The formatting looks odd and the indentation doesn't really line up well with the code examples and sometimes present subsequent non-bulleted supplemental sentences (e.g. about how to clear `Timeout`/`Interval` objects).